pkg/storage/article_storage.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"news-aggregator/pkg/model"
+	"time"
+)
+
+//go:generate mockgen -destination=mocks/mock_article.go -package=mocks news-aggregator/pkg/storage ArticleStorage
+
+// ArticleStorage is an interface that defines the methods for interacting with the article storage.
+type ArticleStorage interface {
+	GetAll() ([]model.Article, error)
+	Create(article model.Article) (model.Article, error)
+	SaveAll(articles []model.Article) error
```

Comment:
Create and SaveAll. I would rename it to `Create` and `CreateAll` or `Save` and `SaveAll`